saleor/plugins/webhook/tasks.py

```diff
@@ -130,8 +132,11 @@ def trigger_all_webhooks_sync(
     this function returns None.
     """
     webhooks = _get_webhooks_for_event(event_type)
-    event_payload = EventPayload.objects.create(payload=data)
+    event_payload = None
     for webhook in webhooks:
+        event_payload = event_payload or EventPayload.objects.create(
```

Comment:
Why do you create EventPayload for each webhook? Is it expected? If yes, why don't you use bulk action here?

Reply:
If the payload is the same there should be only one EventPayload object created in the database.